### ISSUE TITLE
Alias Names do not display if there are more than 1 value in a Stat Panel

### ISSUE THREAD

**Issue by viper25** [grafana/grafana#25132]
**What happened**: Alias Names do not display if there are more than 1 value in a Stat Panel

**What you expected to happen**: If the stat Panel has more ONLY 1 item it ought to display the alias name. I've used the variable {{label}} for the Alias.

**How to reproduce it (as minimally and precisely as possible)**:
Create a New Dashboard with a CloudWatch Data Source. QueryMode: CLoudWatch Metrics with these settings:
![image](https://user-images.githubusercontent.com/327990/82978322-e16d9000-a016-11ea-8cc4-df82c8d1d04d.png)

In fact any metric that shows more than 1 item. I chose WriteThrottleEvens. As you can see the Alias Name is not rendered because there's only ONE value. Now I increase the timeline (say, to 3 hrs to have more than 1 table appear here) and now the table names appear (I've blurred it but the actual table names appear):
![image](https://user-images.githubusercontent.com/327990/82978458-314c5700-a017-11ea-8617-4230782293f3.png)

So basically if it's 1 table no table names appear but it does if it's multiple.

**Anything else we need to know?**:

Not sure if it's related, but if I change the "Display Name", the names of the tables are replaced. Is this intended behaviour? 

![image](https://user-images.githubusercontent.com/327990/82978518-5d67d800-a017-11ea-860f-afc238fe1b0b.png)

**Environment**:
- Grafana version:v7.0.0 (aee1438ff2)
- Data source type & version:CloudWatch
- OS Grafana is installed on:Ubuntu
- User OS & Browser:Windows 10, Chrome Version 81.0.4044.138 (Official Build) (64-bit)
- Grafana plugins: None
- Others:

**Comment by torkelo** [comment]
This is the default behavior in the stat panel, it only shows the display name if more than 1 series/field, or if manually set. 

You can set display name to `${__field.name}` to force it to show even when there is only one field/series

**Comment by viper25** [comment]
Thank you. `${__field.name}` helps. Where in the documentation can I find variables like these to be used?

**Comment by MikeC-BC** [comment]
Very curious about this too. Can tags be used in the display name, if so, how?
